jib-gradle-plugin/README.md

```diff
@@ -161,6 +161,7 @@ Field | Type | Default | Description
 `to` | [`to`](#to-closure) | *Required* | Configures the target image to build your application to.
 `from` | [`from`](#from-closure) | See [`from`](#from-closure) | Configures the base image to build your application on top of.
 `container` | [`container`](#container-closure) | See [`container`](#container-closure) | Configures the container that is run from your built image.
+`extraDirectory` | [`extraDirectory`](#extradirectory-closure) / `File` | See `extraDirectory` | Configures the directory used to add arbitrary files to the image.
```

Comment:
Isn't default always project_dir/src/main/jib?

Reply:
Oh yeah, I meant to specify that in the `extraDirectory` table but deleted it in favor of the minimal table layout.